Lab_5/CountriesRecommendation/CountriesRecommendation/CountrySizeParse.cs

```diff
@@ -0,0 +1,21 @@
+namespace CountriesRecommendation
+{
+    public class CountrySizeParse
```

Comment:
Супер! Теперь у нас есть отдельный метод, который отвечает за парсинг строки в значение `CountrySize` :+1:   

Класс лучше назвать `CountrySizeParser`, так как Parse - это глагол (парсить/разбирать), а тот класс, который предоставляет этот метод, называется существительным Parser (парсер/разборщик).  

Кстати, в случае, когда класс имеет только статические методы, то нет смысла создавать его объет, а следовательно, вес класс можно сделать статическим
```suggestion
    public static class CountrySizeParse
```